SMLHelper/Patchers/OptionsPanelPatcher.cs

```diff
@@ -11,17 +12,19 @@ internal class OptionsPanelPatcher
 
         internal static void Patch(HarmonyInstance harmony)
         {
-            var uGUI_OptionsPanelType = typeof(uGUI_OptionsPanel);
-            var thisType = typeof(OptionsPanelPatcher);
-            var startMethod = uGUI_OptionsPanelType.GetMethod("AddTabs", BindingFlags.NonPublic | BindingFlags.Instance);
+            Type uGUI_OptionsPanelType = typeof(uGUI_OptionsPanel);
+            Type thisType = typeof(OptionsPanelPatcher);
+            MethodInfo startMethod = uGUI_OptionsPanelType.GetMethod("AddTabs", BindingFlags.NonPublic | BindingFlags.Instance);
 
             harmony.Patch(startMethod, null, new HarmonyMethod(thisType.GetMethod("AddTabs_Postfix", BindingFlags.NonPublic | BindingFlags.Static)));
         }
 
         internal static void AddTabs_Postfix(uGUI_OptionsPanel __instance)
         {
-            var optionsPanel = __instance;
-            var modsTab = optionsPanel.AddTab("Mods");
+            uGUI_OptionsPanel optionsPanel = __instance;
+            int modsTab = optionsPanel.AddTab("Mods");
+
+            if (modOptions.Count <= 0) optionsPanel.AddHeading(modsTab, "No options here...");
```

Comment:
This won't work if mods outside of SMLHelper add mod options in the mod options tab. Better solution is to use reflection or something else, to see if anything exists in the mods tab.